massa-final-state/src/executed_ops.rs

```diff
@@ -55,6 +56,147 @@ impl ExecutedOps {
         self.0
             .retain(|_id, last_valid_slot| *last_valid_slot >= max_slot);
     }
+
+    /// Get a part of the executed operations.
+    ///
+    /// Solely used by the bootstrap.
+    ///
+    /// # Returns
+    /// A tuple containing the data and the next executed ops streaming step
+    pub fn get_executed_ops_part(
+        &self,
+        cursor: ExecutedOpsStreamingStep,
+    ) -> Result<(Vec<u8>, ExecutedOpsStreamingStep), ModelsError> {
+        // TODO: stream in multiple parts
+        match cursor {
+            ExecutedOpsStreamingStep::Started => (), // TODO: when parts start at unbounded left range
+            ExecutedOpsStreamingStep::Ongoing(_op_id) => (), // TODO: when parts start at op_id left range
+            ExecutedOpsStreamingStep::Finished => {
+                return Ok((Vec::new(), ExecutedOpsStreamingStep::Finished))
```

Comment:
what happens if we have finished, but new changes appear later ? will we ignore the incoming changes ?